Proposal: Resource Tags Working Group (#2837)
open-telemetry/community: projects/service-and-deployment-semconv.md

> # Service and Deployment Semconv SIG

Reviewer: ```suggestion
# Service and Deployment Semantic Conventions
```

* Documentation and examples for adoption by cloud providers, observability platforms, and security tools

---
## Staffing / Help Wanted

We are seeking domain experts to work on the definition, alignment, and adoption of standardized resource attributes across cloud platforms and observability systems.

The goal is to follow @tedsuo's proposed [Semantic Convention Process](https://docs.google.com/document/d/1ghvajKaipiNZso3fDtyNxU7x1zx0_Eyd02OGpMGEpLE/edit#heading=h.xc2ft2cddhny), with the following stages:

- **Stage 1: Working Group Preparation** — Define scope, gather contributors, and align on the initial set of attributes.
- **Stage 2: Prototyping and Finalizing Semantic Conventions Proposal** — Build prototypes using the proposed attributes and refine the semantic conventions for final review and submission.
- **Stage 3: Implementation** 

---
### Required staffing

**Project Leads:**

- @janhvi31 (Google)

**Domain Experts:**

- @ralf0131 (Alibaba)  
- @horovits (Amazon)  
- @ymotongpoo (Amazon)  
- @reyang (Microsoft)

**Sponsors:**

- @trask (Microsoft) — GC Sponsor  

> - @jsuereth (Google) — TC Sponsor (escalating sponsorship level, pending confirmation)

Reviewer 2: I'll check today.

> - @jsuereth (Google) — TC Sponsor (escalating sponsorship level, pending confirmation)

Reviewer 2: Just confirmed - Yes this only requires escalating sponsorship.

> - @jsuereth (Google) — TC Sponsor (escalating sponsorship level, pending confirmation)

Reviewer: ```suggestion
- @jsuereth (Google) — TC Sponsor (escalating sponsorship level)
```

**Semantic Convention Maintainers:**

- @trask (Microsoft)  
- @jsuereth (Google)
- @joaopgrassi (Dynatrace)
 
---

## Timeline

Stage 1 (Working Group Preparation) is currently underway. We are aligning on the initial scoped set of attributes, gathering contributors, and identifying sponsors and maintainers.

Stage 2 (Stabilizing the Specification) will begin once we have adequate staffing and have aligned on a meeting schedule (currently targeting bi-weekly sessions). 

Stage 3 (Implementation) will begin after the initial tag set is reviewed and marked stable. 

---

## GitHub Project

To be created once proposal is approved.

---

## SIG Meetings and Communication

- **Meeting cadence**: Biweekly

> - **Time**: TBD

Author: Based on everyone's [availability](https://strawpoll.com/2ayLQXNDBn4), we will be setting up two biweekly syncs:

- PT-Friendly: Thursdays @ 8:00 AM PT
- Asia-Friendly: Thursdays @ 5:30 PM IST

I will be present at all meetings to ensure continuity.

> - **Slack**: Proposed new channel `#otel-logical-entity-semconv-sig` on CNCF Slack

Reviewer: ```suggestion
- **Slack**: Proposed new channel `#otel-service-and-deployment-semconv-sig` on CNCF Slack
```
